src/main/java/Duke.java

```diff
@@ -1,10 +1,282 @@
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static final ArrayList<String> Data = new ArrayList<>();
+    private static boolean hasData = false;
+    public static void main(String[] args) throws FileNotFoundException {
+        String LINE  = "_______________________________________________________________\n";
+
+
+
+        Scanner in = new Scanner(System.in);
+
+        //Task[] Task = new Task[100];
+        ArrayList<Task> Task = new ArrayList<>();
+
+        String userInput;
+        String date = "";
+
+        int count = 0;
+
+        String fileName = "C:\\data\\duke.txt"; //handles data file creation
```

Comment:
Consider writing comments above the code, relative to the position of the code, as per coding standards.